src/main/java/goal/tools/PlatformManager.java

```diff
@@ -480,12 +475,15 @@ public MASProgram parseMASFile(File masFile) throws ParserException, FileNotFoun
 			// Provide message to logger.
 			parserLogger.log(new StringsLogRecord(Level.INFO, message
 					.toString()));
-
-			// Parse the agent files that are part of the MAS file.
-			parseGOALFiles(masProgram);
-
-			return masProgram;
 		}
+		
+		// parse any found agent files anyway
+		parseGOALFiles(masProgram);
+		
+		// parse emotion config if found
+		parseEmotionConfig(masProgram);
```

Comment:
De emotionconfig parsen in de platformmanager werkten volgens mij niet altijd (wordt niet door alle manieren van files runnen gecalled), daarom dat hij nu in AbstractRun wordt gedaan.